internal/configuration/settings/helpers/copy.go

```diff
@@ -103,16 +103,6 @@ func CopyIPNet(original net.IPNet) (copied net.IPNet) {
 	return copied
 }
 
-func CopyIPNetPtr(original *net.IPNet) (copied *net.IPNet) {
-	if original == nil {
-		return nil
-	}
-
-	copied = new(net.IPNet)
-	*copied = CopyIPNet(*original)
-	return copied
-}
-
```

Comment:
Let's keep this here. I understand it's unused, but I left it there since it should be used soon (for DNS).